pandas/core/dtypes/concat.py

```diff
@@ -604,12 +603,8 @@ def _concat_rangeindex_same_dtype(indexes):
         # Get the stop value from "next" or alternatively
         # from the last non-empty index
         stop = non_empty_indexes[-1]._stop if next is None else next
+        return RangeIndex(start, stop, step)
     else:
         # Here all "indexes" had 0 length, i.e. were empty.
-        # Simply take start, stop, and step from the last empty index.
-        obj = indexes[-1]
-        start = obj._start
-        step = obj._step
-        stop = obj._stop
-
-    return indexes[0].__class__(start, stop, step)
+        # In this case return an empty integer index.
```

Comment:
I think that this should be ``RangeIndex(0, 1)`` to be consistent with our default indices, e.g. ``Series([]).index``

Reply:
`RangeIndex(0, 1)` or rather `RangeIndex(0, 0)`?

Reply:
sorry yes

Reply:
it is changed to empty range